clients/web/src/stylesheets/widgets/accessWidget.styl

```diff
@@ -28,6 +28,21 @@
   font-weight bold
   padding 3px 10px
 
+i.g-flag-icon
+  position relative
+
+.g-flag-count-indicator
+  display inline-block
+  border-radius 3px
+  background-color rgba(200, 0, 0, 0.75)
```

Comment:
In case the dot was covering up the icon, I figured it might be helpful to show the shape of the icon behind. Might not look good, though. Should I change it?